### ISSUE TITLE
Configuration issue on Custom Learning Admin page

### ISSUE THREAD

**Issue by livinusb** [pnp/custom-learning-office-365#608]
>**PROVISIONING NOTE**: If you are having issues with the SharePoint look book while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the SharePoint look book to [SharePoint/sp-provisioning-service GitHub Issues List](https://github.com/SharePoint/sp-provisioning-service/issues).

>**WEB PART NOT LOADING**: If you are getting the error: "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". Please review the [Diagnosing Pre-Check](https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.md) guidance before submitting your bug.

## Describe the bug

Admittedly, I had been tweaking some permissions on the site when I started to experience the bug specifically for the customlearningadmin page as listed below:

Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance. [Administrators: Please see the browser console for detailed logs. For technical assistance check out the issues list at https://github.com/pnp/custom-learning-office-365/issues.] {CustomLearningAdminWebPart}

custom-learning-admin-web-part_a856afe0e07b13cfeb87.js:1 
 Message: 🎓 M365LP:InitService (loadUserRole) - TypeError: Cannot read properties of undefined (reading 'toString')

### To Reproduce

Reproduce by trying to access to the custom learning admin page

### Expected behavior

To be able to access the custom learning admin page.

### Screenshots

![image](https://user-images.githubusercontent.com/5559176/141811480-e1cddf1c-59d8-449d-a191-3f665646edcf.png)
![image](https://user-images.githubusercontent.com/5559176/141811634-cbc2696c-f71f-4cd9-a9dc-36f6f00d9a8d.png)

### Learning Pathways version number
4.14 on user end. After updating app in app catalog, we were unable to update learning pathways on the MLP site. 
4.30 in tenant app catalog

>Please share the version number of learning pathways in your tenant app catalog.
 
### Additional context

Would it be best to delete and redeploy learning pathway? If so, how should we go about doing so?

**Comment by amitkh22** [comment]
Hi @livinusb, Thanks for submitting this issue. Please allow us sometime to investigate. We'll get back to you at the earliest. Thank you for your patience.

**Comment by dcashpeterson** [comment]
@livinusb thanks for reaching out. The function that is erroring out is the one that checks to make sure that the user is part of the owners or member group so most likely if you were changing permissions the user is no longer in one of the associated groups. I would suggest that you put the permissions back and see if you can get things working again. 

Once that is done check to make sure that the app was deployed to the tenant app catalog with no errors. The error field is off to the right so you might need to scroll. You shouldn't need to delete it from the app catalog. You can try to upload the 4.3 sppkg file to the app catalog again and ensure there are no errors. In the learning pathways site collection you should be able to see app and in the details there you can upgrade it. Keep in mind, the user doing the upgrade needs to have permissions to do so or it will be grayed out. 

Let me know if any of these comes up with any additional info. 

-Derek

**Comment by livinusb** [comment]
Hi @dcashpeterson. Thanks for getting back to me on this issue. 

I had deleted the member's group and re-created it. That might be the issue. I'm currently part of the owners and member group.

I don't see the option to upgrade it at all: 
![image](https://user-images.githubusercontent.com/5559176/142029776-91b0a73c-8528-452f-9249-4a2288dcddb9.png)

I also just confirmed with our SharePoint admin that he might have deployed it via the SharePoint look book, so I'm probably barking up the wrong tree.
